### DISCUSSION TITLE
Migration of Retail SDK project to Commerce SDK

### DISCUSSION THREAD

**Discussion by Riaz-Dynamics365FOAndCommerce** [microsoft/Dynamics365Commerce.Solutions#190] [Q&A]
Dear All

I have downloaded samples from GitHub and successfully built in VS Code.
Now I have to Migrate my retail SDK project to Commerce SDK and from documentation, I am not able to start process. Is there anyone who can help me either by screen sharing session or by sharing relevant videos. If anybody who has experience comes with me in screen sharing session, I will pay for it. Kindly help me because I am stuck now.

**Comment by lukedgr** [comment]
Hi @JPLRiaz , unfortunately, the Microsoft product team isn't able to do screen sharing sessions, but there may be others in the community who are able to help. Until then, I hope you'll find the resources below helpful to get started or if you have any more specific questions I'm happy to help in this forum.

[Store Commerce Migration Series](https://community.dynamics.com/blogs/post/?postid=e7344989-6527-4a83-b74e-be5e670b66ee)
[Commerce SDK Extensions Deep Dive](https://community.dynamics.com/blogs/post/?postid=a7ae4e0b-3af0-48a0-8943-9ee9d0f941c6)
[POS Extension Migration Document](https://learn.microsoft.com/en-us/dynamics365/commerce/dev-itpro/pos-extension/migrate-pos-extension)
[Retail SDK to Commerce SDK Migration Document](https://learn.microsoft.com/en-us/dynamics365/commerce/dev-itpro/retail-sdk/migrate-commerce-sdk)
